source/navitiacommon/navitiacommon/models.py

```diff
@@ -263,6 +265,45 @@ def __repr__(self):
         return '<Instance %r>' % self.name
 
 
+class TravelerProfile(db.Model):
+
+    coverage = db.Column(db.Text, db.ForeignKey('instance.name'), primary_key=True, nullable=True)
+    traveler_type = db.Column('traveler_type',
+                              db.Enum('standard', 'slow_walker', 'fast_walker', 'luggage',
+                                       'wheelchair', 'cyclist', 'motorist', name='traveler_type'),
+                              default='standard',
+                              primary_key=True, nullable=False)
+    __table_args__ = (db.UniqueConstraint('coverage', 'traveler_type'), )
```

Comment:
I did something wrong... coverage shouldn't be primary and unique. the composited key('coverage', 'traveler_type') should be unique. 
 We may want something like ('default_id', 'slow_walker'), ('default_id', 'standard'), ('idf', 'standard')